.evergreen/install-dependencies.sh

```diff
@@ -91,6 +91,7 @@ EOT
   nvm use "$NODE_VERSION"
   which node || echo "node not found, PATH=$PATH"
   which npm || echo "npm not found, PATH=$PATH"
+  npm cache clear --force # Fixes: Cannot read properties of null (reading 'pickAlgorithm') error on windows
```

Comment:
Could this interfere with the optional dependencies that we rely on to be automatically installed? Have you seen the new failures on this PR?